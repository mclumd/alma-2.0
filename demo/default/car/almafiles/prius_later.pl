obs(quote(rel(car, prius))).
obs(quote(rel(charged, prius))).
obs(quote(not(rel(fueled, prius)))).

fif(now(3), obs(quote(rel(electric_car, prius)))).
fif(now(3), obs(quote(rel(gas_tank_car, prius)))).
