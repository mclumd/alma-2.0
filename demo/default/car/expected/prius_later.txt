rel(car, prius)
rel(gas_tank_car, prius)
rel(electric_car, prius)
rel(charged, prius)
~rel(fueled, prius)
rel(powered, prius)
abnormal(prius, gas_tank_car, "rel(powered, prius)")
